image_processing/src/app.py

```diff
@@ -0,0 +1,100 @@
+from h2o_wave import Q, app, ui, main
+import ip_utils as ip
+import ui_utils as U
+import helper
+import cv2
+import os
+import pandas as pd
+import views.layout_utils as layouts
+
+
+@app('/ip')
+async def serve(q: Q):
+    _hash = q.args['#']
+
+    if not q.client.flag:
```

Comment:
The variable name `flag` is not very descriptive, which makes it hard for new users to know what this specific code is checking or doing. Could it be renamed more descriptive, or add a comment as to what the `flag` means and what this if statement is doing? Maybe `q.client.app_initialized` or something like that

Reply:
Actually, what is the difference between the `q.client.flag` check and the `q.client.initialized` check? Should these be merged?